menuinst/platforms/base.py

```diff
@@ -219,8 +219,16 @@ def _paths(self) -> Iterable[os.PathLike]:
     def _initialize_on_defaults(data) -> Dict:
         with open(data_path(f"menuinst-{SCHEMA_VERSION}.default.json")) as f:
             defaults = json.load(f)["menu_items"][0]
-
-        return deep_update(defaults, data)
+        original = deepcopy(data.copy())
+        data = deep_update(defaults, data)
+        # remove platforms that were not originally in 'data'
+        data["platforms"] = {
+            platform: value
+            for platform, value in data.get("platforms", {}).items()
+            if platform in original.get("platforms", {})
+            and original.get("platforms", {}).get(platform) not in (None, False)
```

Comment:
```suggestion
            if original.get("platforms", {}).get(platform) not in (None, False)
```

The first part of the if-condition seems redundant, doesn't it?